ticdc/ticdc-sink-to-mysql.md

```diff
@@ -36,7 +36,7 @@ Info: {"sink-uri":"mysql://root:123456@127.0.0.1:3306/","opts":{},"create-time":
 Sink URI 用于指定 TiCDC 目标系统的连接信息，遵循以下格式：
 
 ```
-[scheme]://[userinfo@][host]:[port][/path]?[query_parameters]
+[schema]://[userinfo@][host]:[port][/path]?[query_parameters]
```

Comment:
Scheme follows the uri syntax specified in rfc3986, ref: https://datatracker.ietf.org/doc/html/rfc3986#section-3

Reply:
wow, got it.🤣